snuba/templates/query.html

```diff
@@ -2,7 +2,10 @@
 {% block content %}
     <script>
     function run() {
-      fetch('/query', {
+      dataset = window.location.pathname.split("/").filter(function(p){ return p !== "" })[0]
```

Comment:
does this still work on events if you open /query without passing a dataset in the URL ?

Reply:
Yah. It redirects to `/events/query`.

Reply:
I had more conditionals before that would check for that case but given that we're redirecting, this felt cleaner.

Reply:
Tried to add a better comment earlier, but internet: can't the URL passed here just be the pathname itself? I don't think any URL parsing needs to happen here.

Reply:
Ooo that's even better. Change coming!